images/win/scripts/Tests/Android.Tests.ps1

```diff
@@ -2,16 +2,21 @@ Import-Module (Join-Path $PSScriptRoot "..\SoftwareReport\SoftwareReport.Android
 
 Describe "Android SDK" {
     $androidToolset = (Get-ToolsetContent).android
+    $androidPackages = Get-AndroidPackages
     $androidInstalledPackages = Get-AndroidInstalledPackages
 
     $platformTestCases = @()
-    $platformList = $androidToolset.platform_list
+    [int]$platformMinVersion = $androidToolset.platform_min_version
+    $platformList = $androidPackages | Where-Object { "$_".StartsWith("platforms;") } |
+        Where-Object { [int]$_.Split("-")[1] -ge $platformMinVersion } | Sort-Object { [int]$_.Split("-")[1] } -Unique
```

Comment:
May be create some helper function that can be reused by installer script and test?